### ISSUE TITLE
Error on build from aur

### ISSUE THREAD

**Issue by BjornMelgaard** [jwilm/alacritty#402]
Excerpt
```
   Compiling inotify v0.2.3
   Compiling font v0.1.0 (file:///tmp/yaourt-tmp-bjorn/aur-alacritty-git/src/alacritty-git/font)
error[E0432]: unresolved import `self::ffi::FcConfigGetCurrent`
  --> font/src/ft/list_fonts.rs:27:21
   |
27 |     use self::ffi::{FcConfigGetCurrent, FcConfigGetFonts, FcSetSystem, FcSetApplication};
   |                     ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcConfigGetFonts`
  --> font/src/ft/list_fonts.rs:27:41
   |
27 |     use self::ffi::{FcConfigGetCurrent, FcConfigGetFonts, FcSetSystem, FcSetApplication};
   |                                         ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcSetSystem`
  --> font/src/ft/list_fonts.rs:27:59
   |
27 |     use self::ffi::{FcConfigGetCurrent, FcConfigGetFonts, FcSetSystem, FcSetApplication};
   |                                                           ^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcSetApplication`
  --> font/src/ft/list_fonts.rs:27:72
   |
27 |     use self::ffi::{FcConfigGetCurrent, FcConfigGetFonts, FcSetSystem, FcSetApplication};
   |                                                                        ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternGetString`
  --> font/src/ft/list_fonts.rs:28:21
   |
28 |     use self::ffi::{FcPatternGetString, FcPatternCreate, FcPatternAddString};
   |                     ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternCreate`
  --> font/src/ft/list_fonts.rs:28:41
   |
28 |     use self::ffi::{FcPatternGetString, FcPatternCreate, FcPatternAddString};
   |                                         ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternAddString`
  --> font/src/ft/list_fonts.rs:28:58
   |
28 |     use self::ffi::{FcPatternGetString, FcPatternCreate, FcPatternAddString};
   |                                                          ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternGetInteger`
  --> font/src/ft/list_fonts.rs:29:21
   |
29 |     use self::ffi::{FcPatternGetInteger, FcPatternAddInteger};
   |                     ^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternAddInteger`
  --> font/src/ft/list_fonts.rs:29:42
   |
29 |     use self::ffi::{FcPatternGetInteger, FcPatternAddInteger};
   |                                          ^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcObjectSetCreate`
  --> font/src/ft/list_fonts.rs:30:21
   |
30 |     use self::ffi::{FcObjectSetCreate, FcObjectSetAdd};
   |                     ^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcObjectSetAdd`
  --> font/src/ft/list_fonts.rs:30:40
   |
30 |     use self::ffi::{FcObjectSetCreate, FcObjectSetAdd};
   |                                        ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcResultMatch`
  --> font/src/ft/list_fonts.rs:31:21
   |
31 |     use self::ffi::{FcResultMatch, FcResultNoMatch, FcFontSetList};
   |                     ^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcResultNoMatch`
  --> font/src/ft/list_fonts.rs:31:36
   |
31 |     use self::ffi::{FcResultMatch, FcResultNoMatch, FcFontSetList};
   |                                    ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontSetList`
  --> font/src/ft/list_fonts.rs:31:53
   |
31 |     use self::ffi::{FcResultMatch, FcResultNoMatch, FcFontSetList};
   |                                                     ^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcChar8`
  --> font/src/ft/list_fonts.rs:32:21
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                     ^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcConfig`
  --> font/src/ft/list_fonts.rs:32:30
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                              ^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPattern`
  --> font/src/ft/list_fonts.rs:32:40
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                                        ^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontSet`
  --> font/src/ft/list_fonts.rs:32:51
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                                                   ^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcObjectSet`
  --> font/src/ft/list_fonts.rs:32:62
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                                                              ^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcCharSet`
  --> font/src/ft/list_fonts.rs:32:75
   |
32 |     use self::ffi::{FcChar8, FcConfig, FcPattern, FcFontSet, FcObjectSet, FcCharSet};
   |                                                                           ^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontSetDestroy`
  --> font/src/ft/list_fonts.rs:33:21
   |
33 |     use self::ffi::{FcFontSetDestroy, FcPatternDestroy, FcObjectSetDestroy, FcConfigDestroy};
   |                     ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcPatternDestroy`
  --> font/src/ft/list_fonts.rs:33:39
   |
33 |     use self::ffi::{FcFontSetDestroy, FcPatternDestroy, FcObjectSetDestroy, FcConfigDestroy};
   |                                       ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcObjectSetDestroy`
  --> font/src/ft/list_fonts.rs:33:57
   |
33 |     use self::ffi::{FcFontSetDestroy, FcPatternDestroy, FcObjectSetDestroy, FcConfigDestroy};
   |                                                         ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcConfigDestroy`
  --> font/src/ft/list_fonts.rs:33:77
   |
33 |     use self::ffi::{FcFontSetDestroy, FcPatternDestroy, FcObjectSetDestroy, FcConfigDestroy};
   |                                                                             ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontMatch`
  --> font/src/ft/list_fonts.rs:34:21
   |
34 |     use self::ffi::{FcFontMatch, FcFontList, FcFontSort, FcConfigSubstitute, FcDefaultSubstitute};
   |                     ^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontList`
  --> font/src/ft/list_fonts.rs:34:34
   |
34 |     use self::ffi::{FcFontMatch, FcFontList, FcFontSort, FcConfigSubstitute, FcDefaultSubstitute};
   |                                  ^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcFontSort`
  --> font/src/ft/list_fonts.rs:34:46
   |
34 |     use self::ffi::{FcFontMatch, FcFontList, FcFontSort, FcConfigSubstitute, FcDefaultSubstitute};
   |                                              ^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcConfigSubstitute`
  --> font/src/ft/list_fonts.rs:34:58
   |
34 |     use self::ffi::{FcFontMatch, FcFontList, FcFontSort, FcConfigSubstitute, FcDefaultSubstitute};
   |                                                          ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcDefaultSubstitute`
  --> font/src/ft/list_fonts.rs:34:78
   |
34 |     use self::ffi::{FcFontMatch, FcFontList, FcFontSort, FcConfigSubstitute, FcDefaultSubstitute};
   |                                                                              ^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcMatchFont`
  --> font/src/ft/list_fonts.rs:35:21
   |
35 |     use self::ffi::{FcMatchFont, FcMatchPattern, FcMatchScan, FC_SLANT_ITALIC, FC_SLANT_ROMAN};
   |                     ^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcMatchPattern`
  --> font/src/ft/list_fonts.rs:35:34
   |
35 |     use self::ffi::{FcMatchFont, FcMatchPattern, FcMatchScan, FC_SLANT_ITALIC, FC_SLANT_ROMAN};
   |                                  ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FcMatchScan`
  --> font/src/ft/list_fonts.rs:35:50
   |
35 |     use self::ffi::{FcMatchFont, FcMatchPattern, FcMatchScan, FC_SLANT_ITALIC, FC_SLANT_ROMAN};
   |                                                  ^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_SLANT_ITALIC`
  --> font/src/ft/list_fonts.rs:35:63
   |
35 |     use self::ffi::{FcMatchFont, FcMatchPattern, FcMatchScan, FC_SLANT_ITALIC, FC_SLANT_ROMAN};
   |                                                               ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_SLANT_ROMAN`
  --> font/src/ft/list_fonts.rs:35:80
   |
35 |     use self::ffi::{FcMatchFont, FcMatchPattern, FcMatchScan, FC_SLANT_ITALIC, FC_SLANT_ROMAN};
   |                                                                                ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_SLANT_OBLIQUE`
  --> font/src/ft/list_fonts.rs:36:21
   |
36 |     use self::ffi::{FC_SLANT_OBLIQUE};
   |                     ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_THIN`
  --> font/src/ft/list_fonts.rs:37:21
   |
37 |     use self::ffi::{FC_WEIGHT_THIN, FC_WEIGHT_EXTRALIGHT, FC_WEIGHT_LIGHT};
   |                     ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_EXTRALIGHT`
  --> font/src/ft/list_fonts.rs:37:37
   |
37 |     use self::ffi::{FC_WEIGHT_THIN, FC_WEIGHT_EXTRALIGHT, FC_WEIGHT_LIGHT};
   |                                     ^^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_LIGHT`
  --> font/src/ft/list_fonts.rs:37:59
   |
37 |     use self::ffi::{FC_WEIGHT_THIN, FC_WEIGHT_EXTRALIGHT, FC_WEIGHT_LIGHT};
   |                                                           ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_BOOK`
  --> font/src/ft/list_fonts.rs:38:21
   |
38 |     use self::ffi::{FC_WEIGHT_BOOK, FC_WEIGHT_REGULAR, FC_WEIGHT_MEDIUM, FC_WEIGHT_SEMIBOLD};
   |                     ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_REGULAR`
  --> font/src/ft/list_fonts.rs:38:37
   |
38 |     use self::ffi::{FC_WEIGHT_BOOK, FC_WEIGHT_REGULAR, FC_WEIGHT_MEDIUM, FC_WEIGHT_SEMIBOLD};
   |                                     ^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_MEDIUM`
  --> font/src/ft/list_fonts.rs:38:56
   |
38 |     use self::ffi::{FC_WEIGHT_BOOK, FC_WEIGHT_REGULAR, FC_WEIGHT_MEDIUM, FC_WEIGHT_SEMIBOLD};
   |                                                        ^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_SEMIBOLD`
  --> font/src/ft/list_fonts.rs:38:74
   |
38 |     use self::ffi::{FC_WEIGHT_BOOK, FC_WEIGHT_REGULAR, FC_WEIGHT_MEDIUM, FC_WEIGHT_SEMIBOLD};
   |                                                                          ^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_BOLD`
  --> font/src/ft/list_fonts.rs:39:21
   |
39 |     use self::ffi::{FC_WEIGHT_BOLD, FC_WEIGHT_EXTRABOLD, FC_WEIGHT_BLACK, FC_WEIGHT_EXTRABLACK};
   |                     ^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_EXTRABOLD`
  --> font/src/ft/list_fonts.rs:39:37
   |
39 |     use self::ffi::{FC_WEIGHT_BOLD, FC_WEIGHT_EXTRABOLD, FC_WEIGHT_BLACK, FC_WEIGHT_EXTRABLACK};
   |                                     ^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_BLACK`
  --> font/src/ft/list_fonts.rs:39:58
   |
39 |     use self::ffi::{FC_WEIGHT_BOLD, FC_WEIGHT_EXTRABOLD, FC_WEIGHT_BLACK, FC_WEIGHT_EXTRABLACK};
   |                                                          ^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0432]: unresolved import `self::ffi::FC_WEIGHT_EXTRABLACK`
  --> font/src/ft/list_fonts.rs:39:75
   |
39 |     use self::ffi::{FC_WEIGHT_BOLD, FC_WEIGHT_EXTRABOLD, FC_WEIGHT_BLACK, FC_WEIGHT_EXTRABLACK};
   |                                                                           ^^^^^^^^^^^^^^^^^^^^ Could not find `ffi` in `ft::list_fonts::fc`

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:274:18
    |
274 |         Italic = FC_SLANT_ITALIC as isize,
    |                  ^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:275:19
    |
275 |         Oblique = FC_SLANT_OBLIQUE as isize,
    |                   ^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:276:17
    |
276 |         Roman = FC_SLANT_ROMAN as isize,
    |                 ^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:281:16
    |
281 |         Thin = FC_WEIGHT_THIN as isize,
    |                ^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:282:22
    |
282 |         Extralight = FC_WEIGHT_EXTRALIGHT as isize,
    |                      ^^^^^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:283:17
    |
283 |         Light = FC_WEIGHT_LIGHT as isize,
    |                 ^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:284:16
    |
284 |         Book = FC_WEIGHT_BOOK as isize,
    |                ^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:285:19
    |
285 |         Regular = FC_WEIGHT_REGULAR as isize,
    |                   ^^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:286:18
    |
286 |         Medium = FC_WEIGHT_MEDIUM as isize,
    |                  ^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:287:20
    |
287 |         Semibold = FC_WEIGHT_SEMIBOLD as isize,
    |                    ^^^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:288:16
    |
288 |         Bold = FC_WEIGHT_BOLD as isize,
    |                ^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:289:21
    |
289 |         Extrabold = FC_WEIGHT_EXTRABOLD as isize,
    |                     ^^^^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:290:17
    |
290 |         Black = FC_WEIGHT_BLACK as isize,
    |                 ^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:291:22
    |
291 |         Extrablack = FC_WEIGHT_EXTRABLACK as isize,
    |                      ^^^^^^^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:210:16
    |
210 |         Font = FcMatchFont as isize,
    |                ^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:211:19
    |
211 |         Pattern = FcMatchPattern as isize,
    |                   ^^^^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:212:16
    |
212 |         Scan = FcMatchScan as isize,
    |                ^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:203:18
    |
203 |         System = FcSetSystem as isize,
    |                  ^^^^^^^^^^^ non-constant path in constant expression

error[E0080]: constant evaluation error
   --> font/src/ft/list_fonts.rs:204:23
    |
204 |         Application = FcSetApplication as isize,
    |                       ^^^^^^^^^^^^^^^^ non-constant path in constant expression

error: aborting due to 19 previous errors

```

**Comment by jwilm** [comment]
This is an issue with the AUR package out of date. I think it's blocked on Rust 1.15 not being in the AUR yet. Duplicate of #383

**Comment by BjornMelgaard** [comment]
1.14 actually

**Comment by jwilm** [comment]
Alacritty requires 1.15, but only 1.14 is in the AUR.

**Comment by polyzen** [comment]
Just to clarify: Rust is packaged in the repos.

**Comment by BjornMelgaard** [comment]
Yeah, I have installed alacritty after I install rust-nightly-bin, but now it cant find DejaVu Sans Mono for Powerline font :smile: 
As I understand this fonts is not fully supported

[My config](http://pastebin.com/6UYa393M) 
and
```
$ fc-list G -i "DejaVu Sans Mono for Powerline"
/usr/share/fonts/TTF/DejaVu Sans Mono Bold for Powerline.ttf: DejaVu Sans Mono for Powerline:style=Bold
/usr/share/fonts/TTF/DejaVu Sans Mono Oblique for Powerline.ttf: DejaVu Sans Mono for Powerline:style=Oblique
/usr/share/fonts/TTF/DejaVu Sans Mono for Powerline.ttf: DejaVu Sans Mono for Powerline:style=Book
/usr/share/fonts/TTF/DejaVu Sans Mono Bold Oblique for Powerline.ttf: DejaVu Sans Mono for Powerline:style=Bold Oblique
```

**Comment by jwilm** [comment]
You might try specifying "Book" for `font.normal.style`.

> As I understand this fonts is not fully supported

We don't support fallback fonts yet, but that shouldn't be the issue there.

**Comment by BjornMelgaard** [comment]
@jwilm , no, I have tried
Also it don't react on size of font, it big no matter what

```
$ alacritty
num_cols, num_lines = 119, 61
[unhandled osc_dispatch]: [['1',],['.','.','n','f','i','g','/','d','o','t','f','i','l','e','s',],]
```
